docs/content/references/cli/ptb.mdx

```diff
@@ -189,15 +207,26 @@ The following example showcases how to split a gas coin into multiple coins, mak
 --move-call iota::coin::destroy_zero<iota::iota::IOTA> coins.0
 # Can further split a split coin (and through variable bindings/result accesses)
 --split-coins coins.1 [0,0]
---assign zcoins
+--assign zerocoins
 # Destroy both new coins
---move-call iota::coin::destroy_zero<iota::iota::IOTA> zcoins.0
---move-call iota::coin::destroy_zero<iota::iota::IOTA> zcoins.1
-# Can merge the split coins back
+--move-call iota::coin::destroy_zero<iota::iota::IOTA> zerocoins.0
+--move-call iota::coin::destroy_zero<iota::iota::IOTA> zerocoins.1
+# Can merge the split coins back, there is no result, the first coin (gas coin in this example) will have all the coins.
 --merge-coins gas [coins.1, coins.2, coins.3]
 --gas-budget 10000000
 ```
 
+The same commands work for coins that are not `iota::iota::IOTA`. It is just not possible to split them from the gas object, but requires an object ID of the other coin.
+
+```bash
+iota client ptb \
+--assign other_coin_obect_id @0xdab43e6be5f1e3b977448f368c9ec3bb6b9ee75e269633c32d3c94f9f094e23c \
+--split-coins other_coin_obect_id "[0,1,2,3]" \
```

Comment:
```suggestion
--split-coins other_coin_object_id "[0,1,2,3]" \
```

Reply:
Thanks for all the suggestions, merged in https://github.com/iotaledger/iota/pull/4935/commits/b41cb20639b63c707448990de3ecff0cba493500